[2023-02-12 01:20:05,465] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: primeiro_dag.cria_pasta scheduled__2023-02-10T00:00:00+00:00 [queued]>
[2023-02-12 01:20:05,475] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: primeiro_dag.cria_pasta scheduled__2023-02-10T00:00:00+00:00 [queued]>
[2023-02-12 01:20:05,476] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-12 01:20:05,476] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-12 01:20:05,476] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-12 01:20:05,532] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-02-10 00:00:00+00:00
[2023-02-12 01:20:05,535] {standard_task_runner.py:52} INFO - Started process 21395 to run task
[2023-02-12 01:20:05,543] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'primeiro_dag', 'cria_pasta', 'scheduled__2023-02-10T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/var/folders/tf/5c5cm4z11mvb4z72rkp1jvxw0000gn/T/tmpv_ucd7nt', '--error-file', '/var/folders/tf/5c5cm4z11mvb4z72rkp1jvxw0000gn/T/tmpnfe4h699']
[2023-02-12 01:20:05,546] {standard_task_runner.py:80} INFO - Job 5: Subtask cria_pasta
[2023-02-12 01:20:05,609] {task_command.py:370} INFO - Running <TaskInstance: primeiro_dag.cria_pasta scheduled__2023-02-10T00:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2023-02-12 01:20:05,698] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-02-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-10T00:00:00+00:00
[2023-02-12 01:20:05,700] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tf/5c5cm4z11mvb4z72rkp1jvxw0000gn/T
[2023-02-12 01:20:05,701] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p /Users/mikeiasoliveira/Documents/Projetos/etl-crypto/data/pasta_test=2023-02-11T00:00:00+00:00']
[2023-02-12 01:20:05,711] {subprocess.py:85} INFO - Output:
[2023-02-12 01:20:05,802] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-12 01:20:05,847] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=primeiro_dag, task_id=cria_pasta, execution_date=20230210T000000, start_date=20230212T042005, end_date=20230212T042005
[2023-02-12 01:20:06,070] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-12 01:20:06,095] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
